org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java

```diff
@@ -566,7 +567,14 @@ public void onEngineNotRunningEvent(EngineNotRunningEvent event) {
     public void receiveBluetoothStateChanged(BluetoothStateChangedEvent event) {
         // post on decor view to ensure that it gets executed when view has been inflated.
         runAfterInflation(() -> {
-            this.bluetoothIndicator.setEnabled(!event.isBluetoothEnabled);
+            if (!event.isBluetoothEnabled){
+                this.bluetoothIndicator.setBackgroundTintList(ColorStateList.valueOf(getResources().
+                        getColor(R.color.cario_warning_red)));
+            }
+            else {
+                this.bluetoothIndicator.setBackgroundTintList(ColorStateList.valueOf(getResources().
+                        getColor(R.color.cario_color_primary_dark)));
+            }
```

Comment:
Regarding the comment above, maybe it would be sufficient to replace `setEnabled` by `setSelected` and bind the background color within _dashboard_indicator_selector.xml_ to `state_selected`. If it works, do the same for GPS and OBD.

Reply:
Here is the screen recording with `setSelected` . Onclick doesn't work when icons are blue colour. 

https://user-images.githubusercontent.com/70392921/126346601-129d39e9-0fa4-4bd1-bc3f-0d4fd9ad12a2.mp4